static/js/components/TopAppBar_test.js

```diff
@@ -75,13 +76,23 @@ describe("TopAppBar component", () => {
     })
 
     it("has a CatalogMenu component", () => {
+      SETTINGS.enable_course_dropdown = true
       assert.isOk(
         shallow(<TopAppBar currentUser={user} location={null} errorPageHeader={null} courseTopics={courseTopics} />)
           .find("CatalogMenu")
           .exists()
       )
     })
 
+    it("does not have a CatalogMenu component when feature flag is disabled", () => {
+      SETTINGS.enable_course_dropdown = false
+      assert.isNotOk(
+        shallow(<TopAppBar currentUser={user} location={null} errorPageHeader={null} courseTopics={courseTopics} />)
+          .find("CatalogMenu")
+          .exists()
+      )
+    })
+
```

Comment:
I think we can merge this test and the one below with a parameterized test that validates the UI when the flag is on and off.